core/iwasm/include/wasm_export.h

```diff
@@ -713,7 +726,8 @@ wasm_runtime_get_native_addr_range(wasm_module_inst_t module_inst,
   *
   * @return true if success, false otherwise
   */
-bool wasm_runtime_register_natives(const char *module_name,
+WASM_RUNTIME_API_EXTERN bool
+wasm_runtime_register_natives(const char *module_name,
                                    NativeSymbol *native_symbols,
```

Comment:
Indent: after modified, the place of first argument in last line is changed, could you please also change the indent of this line?